cmd/kubeadm/app/master/manifests.go

```diff
@@ -511,17 +506,15 @@ func getSelfHostedAPIServerEnv() []api.EnvVar {
 	return append(getProxyEnvVars(), podIPEnvVar)
 }
 
+// getAuthzParameters gets the authorization-related parameters to the api server
+// At this point, we can assume the list of authorization modes is valid (due to that it has been validated in the API machinery code already)
+// If the list is empty; it's defaulted (mostly for unit testing)
 func getAuthzParameters(modes []string) []string {
 	command := []string{}
 	strset := sets.NewString(modes...)
 
-	// RBAC must always be set, prepend that if not present in the list
-	// TODO(luxas): In v1.8, require the Node and RBAC authorizers to be present
-	// In v1.8 we can utilize the API defaulting/validation to do enforce this,
-	// currently this logic has to be split up into two places, which is sub-optimal.
-	// Cleanup work is possible in and should be done in v1.8
-	if !strset.Has(authzmodes.ModeRBAC) {
-		modes = append([]string{authzmodes.ModeRBAC}, modes...)
+	if len(modes) == 0 {
+		return []string{fmt.Sprintf("--authorization-mode=%s", kubeadmapiext.DefaultAuthorizationModes)}
```

Comment:
depending on how far we get on component config in 1.8 we may want some 3rd object managing "config"